zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py

```diff
@@ -20,6 +21,10 @@ def configure(self, conf):
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
+
+        self._username = conf.get('username', conf.get('user', None))
```

Comment:
why do we have two possible username keys? that's confusing, we should only have one